src/AltinnCore/Common/Services/Implementation/AuthorizationAppSI.cs

```diff
@@ -75,5 +75,28 @@ public List<Party> GetPartyList(int userId)
                 return partyList;
             }
         }
+
+        /// <inheritdoc />
+        public async Task<bool?> ValidateSelectedParty(int userId, int partyId)
```

Comment:
@dskogan @lorang92 alternatively we could get both party and userId from the context. 
I guess the best option would depend on how we envision the endpoint will be used. To verify before allowing someone to select a party, or to validate an action at a later point when the party is already selected. Do you guys have any thoughts?

Reply:
Validation should take place in backend each time user wants to do something on behalf of a party. So we need both.